### PR TITLE
Add migration tool test

### PR DESCRIPTION
**Pull request by tomasdavidorg**
Depends on https://github.com/kiegroup/kie-wb-common/pull/1787

### CODE REVIEW

File: kie-wb-common-cli/kie-wb-common-cli-migration-tool/pom.xml [hunk 1]
```diff
@@ -47,10 +47,37 @@
       <groupId>org.kie.workbench</groupId>
       <artifactId>kie-wb-common-cli-pom-migration</artifactId>
     </dependency>
+    <dependency>
```
Comment (jhrcek) [inline kie-wb-common-cli/kie-wb-common-cli-migration-tool/pom.xml]: Please removed. Every module inherits junit dependency automatically from [here](https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/pom.xml#L2723-L2734)

File: kie-wb-common-cli/kie-wb-common-cli-migration-tool/src/test/java/org/kie/workbench/common/migration/MigrationToolTest.java [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+    @Test
+    public void testNewPomStructure() throws IOException, XmlPullParserException {
+        final File
+                projectDir = getProjectDir(SPACE_B, PROJECT_B2),
+                pom = new File(projectDir, "pom.xml");
+        final MavenXpp3Reader reader = new MavenXpp3Reader();
+        final Model model = reader.read(new FileInputStream(pom));
+
+        assertThat(model.getVersion()).isEqualTo("1.2.3");
+        assertThat(model.getDependencies().size()).isEqualTo(6);
```
Comment (jhrcek) [inline kie-wb-common-cli/kie-wb-common-cli-migration-tool/src/test/java/org/kie/workbench/common/migration/MigrationToolTest.java]: `size()).isEqualTo()` -> `.hasSize(..)`

Ctrl+f for "size())" and apply in whole pr, please.

File: kie-wb-common-cli/kie-wb-common-cli-migration-tool/pom.xml (hunk 1 above)
Comment (tomasdavidorg) [reply kie-wb-common-cli/kie-wb-common-cli-migration-tool/pom.xml]: Fixed. Thanks.

**Comment by tomasdavidorg** [comment]
@desmax74 @pefernan Could you please take a look. 

Depends on #1787 
Cherry pick to 7.7.x: https://github.com/kiegroup/kie-wb-common/pull/1877

**Comment by jhrcek** [comment]
@desmax74 @pefernan can you please take a look / merge this?

**Comment by desmax74** [comment]
The PomMigrationTool.java part is ok

**Comment by tomasdavidorg** [comment]
jenkins retest this

**Comment by tomasdavidorg** [comment]
jenkins retest this

**Comment by tomasdavidorg** [comment]
jenkins retest this

**Comment by tomasdavidorg** [comment]
jenkins retest this

**Comment by tomasdavidorg** [comment]
jenkins retest this

**Comment by tomasdavidorg** [comment]
jenkins retest this

**Comment by tomasdavidorg** [comment]
Finally green. @pefernan Could you please take a look?

**Pull request merged**